tests/pyfunc/test_pyfunc_schema_enforcement.py

```diff
@@ -2659,7 +2659,8 @@ def predict(self, context, model_input, params=None):
         ),
     ],
 )
-def test_pyfunc_model_schema_enforcement_map_type(data, schema):
+@pytest.mark.parametrize("format_key", ["dataframe_split", "dataframe_records"])
```

Comment:
Does 'inputs' work?

Reply:
"inputs" doesn't match this test, it requires the data to be a list.

"dataframe_split", "dataframe_records" should be sufficient in test. We already have 14 child tests in this test.